soc/arm/xilinx_zynqmp/soc.h

```diff
@@ -13,7 +13,6 @@
 #ifndef _ASMLANGUAGE
 
 #include <device.h>
-#include <sys/util.h>
```

Comment:
This change is okay.

In fact, it is getting removed in another PR anyway (#22740).